### DISCUSSION TITLE
네비게이션 비활성화 시 사이드바 활성화 불가능

### DISCUSSION THREAD

**Discussion by TetraTheta** [pronist/hELLO#87] [Ideas]
1400px보다 화면 크기가 작을 경우, 사이드바를 열기 위해서는 네비게이션에 있는 프로필 아이콘을 눌러야 합니다. 하지만 네비게이션을 숨기게 되면 사이드바를 여는 버튼 자체가 사라지므로, 사이드바를 열 수 없게 됩니다.

네비게이션을 비활성화 한 상태에서도 사이드바를 열 수 있는 방법이 필요합니다.

PC에서 확인했습니다. 모바일 환경에서는 확인하지 않았습니다.  
만약 모바일 환경에서 스와이프 동작으로 사이드바를 열 수 있다면, PC 환경까지 고려해 스와이프 이외에도 사이드바를 열 수 있는 방법이 필요합니다.

**Comment by pronist** [comment]
그렇네요. 부분은 개선사항이 될 수 있습니다. :)

**Comment by pronist** [comment]
이 문제에 대해서는 추후 논의가 필요할 듯 싶어 일단 혼란을 줄이기 네비게이션 On/Off 기능을 일시적으로 제거하였습니다. 의도하신 해결방법은 아니겠습니다만 이 부분에 대해서는 추후 개선할 예정입니다 :)
